Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.py

```diff
@@ -0,0 +1,332 @@
+        indicator_metadata['system_service'] = indicator_properties.get('systemService')
+        indicator_metadata['address_prefixes'] = indicator_properties.get('addressPrefixes', [])
+
+        return indicator_metadata
+
+    def extract_indicators_from_values_dict(self, values_from_file: Dict) -> List:
+        """Builds a list of all IP indicators in the input dict.
+
+        Args:
+            values_from_file (Dict): The values object from the Azure downloaded file.
+
+        Returns:
+            list. All indicators that match the filtering options.
+        """
+        results = []
+
+        if values_from_file is None:
+            LOG(F'{INTEGRATION_NAME} - No values in JSON response')
+            return []
+
+        for indicators_group in values_from_file:
+            demisto.debug(F'{INTEGRATION_NAME} - Extracting value: {indicators_group.get("id")}')
+
+            indicator_metadata = self.extract_metadata_of_indicators_group(indicators_group)
+            if not indicator_metadata:
+                continue
+
+            is_region_not_in_filter = 'All' not in self.regions_list and \
+                                      indicator_metadata['region'] not in self.regions_list
+            is_service_not_in_filter = 'All' not in self.services_list and \
+                                       indicator_metadata['system_service'] not in self.services_list
+
+            if is_region_not_in_filter or is_service_not_in_filter:
+                continue
+
+            for address in indicator_metadata['address_prefixes']:
+                results.append(
+                    self.build_ip_indicator(address,
+                                            azure_name=indicator_metadata['name'],
+                                            azure_id=indicator_metadata['id'],
+                                            azure_region=indicator_metadata['region'],
+                                            azure_platform=indicator_metadata['platform'],
+                                            azure_system_service=indicator_metadata['system_service'])
+                )
+
+        return results
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        try:
+            download_link = self.get_azure_download_link()
+            values_from_file = self.get_download_file_content_values(download_link)
+            results = self.extract_indicators_from_values_dict(values_from_file)
+
+            return results
+
+        except (requests.exceptions.SSLError, requests.ConnectionError, requests.exceptions.HTTPError) as err:
+            demisto.debug(str(err))
+            raise Exception(ERROR_TYPE_TO_MESSAGE[err.__class__] + str(err))
+
+        except AttributeError as err:
+            demisto.debug(str(err))
+            raise AttributeError(F'Could not fetch download link from Azure')
```

Comment:
You raise a `RuntimeError` in your code when a download link isn't found... not an `AttributeError`.